R/tidiers_ml_unsupervised_models.R

```diff
@@ -157,7 +157,23 @@ glance.ml_model_gaussian_mixture <- function(x,
                                              ...) {
 
   k <- x$summary$k
-  silhouette <- ml_clustering_evaluator(x$summary$predictions)
-  dplyr::tibble(k = k,
-                silhouette = silhouette)
+  glance_tbl <- dplyr::tibble(k = k)
+  add_silhouette(x, glance_tbl)
+}
+
+# this function add silhouette to glance if
+# spark version is even or greater than 2.3.0
+add_silhouette <- function(x, glance_tbl){
+
+  connection <- spark_connection_find()
```

Comment:
We don't get the connection using `spark_connection_find()` since this would prevent you from using multiple connections, instead, we want to change this code to:

```
sc <- spark_connection(x)
version <- spark_version(sc)
```

`x` for `spark_jobj`, `spark_tbl`, etc. should already have subclassed methods that retrieve the Spark connection from the object, let us know if that's not the case to help out.

Reply:
ok I got it . I'll change  :)